src/test/java/com/intrafind/log4shell/Log4shellUpdateTest.java

```diff
@@ -31,6 +35,8 @@
 
   @Before
   public void before() throws IOException {
+    Log4shellUpdate.IS_WINDOWS = true;
```

Comment:
Shouldn't this be split into windows vs. linux systems? I think it is now always windows

Reply:
It doesn't really make a difference for the tests. It's just a different directory and different line endings (which java doesn't care about anyway).
I can add a separate Linux test.